build/AzurePipelinesTemplates/MUX-RunHelixTests-Job.yml

```diff
@@ -67,6 +74,40 @@ jobs:
       filePath: build\Helix\PrepareHelixPayload.ps1
       arguments: -Platform '$(buildPlatform)' -Configuration '$(buildConfiguration)'
 
+  - template: MUX-CreateHelixProjFile-Steps.yml
+    parameters:
+      condition: ne('${{ parameters.testSuite }}','NugetTestSuite')
+      testFilePath: '$(artifactsDir)\drop\$(buildConfiguration)\$(buildPlatform)\Test\MUXControls.Test.dll'
```

Comment:
Can you explain this condition? What is this parameter for?

Reply:
This YAML file is used both for regular tests and for tests that use the NuGet package.  In the case of regular tests, we want to run tests on MUXControls.Test.dll, MUXControlsTestApp.appx, and IXMPTestApp.appx, whereas for NuGet package we want to use the NuGet tests instead.  We pass different values for this parameter into this template file depending on what tests we want to run.